docs/utilities/recipes.md

```diff
@@ -5,7 +5,7 @@ With the update to Minecraft 1.12, Mojang introduced a new data-driven recipe sy
 
 Loading Recipes
 ---------------
-Forge will load all recipes which can be found within the `./assets/<modid>/recipes/` folder. You can call these files whatever you want, thought the vanilla convention is to name them after the output item. This name is also used as the registration key, but does not affect the operation of the recipe.
+Forge will load all recipes which can be found within the `./data/<modid>/recipes/` folder. You can call these files whatever you want, thought the vanilla convention is to name them after the output item. For multiple recipes from different sources (Smelting, Crafting, etc) one vanilla convention is to use `item_name_from_smelting.json`.  This name is also used as the registration key, but does not affect the operation of the recipe.
```

Comment:
You added an extra space after this sentence.

Reply:
Thanks @tterrag1098 I didn't realize that was a standard.  I'm used to using 2 spaces to separate sentences (Guess I'm old).  I'll remove the space and re-submit.